mappings/net/minecraft/entity/LivingEntity.mapping

```diff
@@ -107,10 +108,12 @@ CLASS ahe net/minecraft/entity/LivingEntity
 	METHOD d getCurrentExperience (Larb;)I
 	METHOD da getActiveItem ()Lawo;
 	METHOD dg isFallFlying ()Z
+	METHOD e turnHead (FF)F
```

Comment:
setHeadRotation? Or what does this do?

Reply:
I couldn't really tell completely. It seemed to have something to do with turning the head (and the body within a certain angle). I have simply removed the mapping as it could be inaccurate.